proposals/0004-nio-redis.md

```diff
@@ -350,22 +351,36 @@ let value = connection.set("my_key", to: 3)
 // value == Optional("14")
 ```
 
+## Metrics
+
+**RedisNIO** comes with basic `SwiftMetrics` integration for the following data points:
+
+- (Counter) # of failed requests
+- (Counter) # of successful requests
+- (Counter) # of connections made
+- (Gauge) # of active connections
+- (Timer) Round-trip time in `ms` to complete a request
+
+They are defined under a `RedisMetrics` struct, and are labeled with the `RedisMetrics.Label` enum for referencing within metric library integrations.
```

Comment:
Perhaps I should add a point mentioning this - but these metrics instances are not configurable by end users. The labels are set to use `"RedisNIO.\(RedisMetrics.Label)"` as their label.

I could see _this_ being a place to perhaps having a URI scheme.